src/NSL/Action/Implementations/hubbardFermiAction.tpp

```diff
@@ -43,14 +43,21 @@ class HubbardFermionAction :
         // inverse temperature
 	    const Type beta;
 
+        // chemical potential
+        const Type mu;
+
         // time slices
         const NSL::size_t Nt;
 
         // lattice
         LatticeType & lattice;
 
         Parameters(const Type & beta, const NSL::size_t & Nt, LatticeType & lattice):
-            beta(beta), Nt(Nt), lattice(lattice), delta(beta/Nt) 
+            beta(beta), mu(0), Nt(Nt), lattice(lattice), delta(beta/Nt) 
+        {}
+
+        Parameters(const Type & beta, const Type & mu, const NSL::size_t & Nt, LatticeType & lattice):
+            beta(beta), mu(mu), Nt(Nt), lattice(lattice), delta(beta/Nt) 
         {}
```

Comment:
We should probably make the order of these parameters match the order of the HubbardExp; see line 47 of [src/NSL/FermionMatrix/Impl/hubbardExp.hpp](https://github.com/Marcel-Rodekamp/NSL/pull/142/files#diff-8d3e3bfb1650c4f28ee599cb5c0b37808ee38698373fdfa86210e84ba1fef382)